### PR TITLE
Storybook main menu

### PR DESCRIPTION
**Pull request by BartDeCaluwe**
This PR will add:

- CredixLogo component that can render a dark or light version of the logo depending on either `prefers-color-scheme` or by passing a prop. 
- WalletButton component that can launch the `@solana/wallet-adapter-react` modal. With a dropdown to: copy address, change wallet, disconnect. 
- MainMenu component that displays the CredixLogo and WalletButton components
- Storybook stories for CredixLogo, WalletButton and MainMenu
- Snapshot test for CredixLogo, WalletButton and MainMenu

Note: The MainMenu Story is not interactive (as in clicking the button does not launch the modal). I'm guessing this is because storybook hasn't been configured to work with our `node_modules`. This is something I'll look into.

### CODE REVIEW

**Comment by BouweCeunen** [comment]
I don't see 2 stories of the CredixLogo on storybook if I click on `Docs`, but I also don't see anything out of the ordinary in the code itself, maybe I'm overlooking something?

**Comment by BartDeCaluwe** [comment]
> I don't see 2 stories of the CredixLogo on storybook if I click on `Docs`, but I also don't see anything out of the ordinary in the code itself, maybe I'm overlooking something?

Not sure what you mean with `Docs`. But the CredixLogo can be seen here: https://c696eb6d.credix-app-v2.pages.dev/?path=/story/credixlogo--light

File: src/components/WalletButton.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { useWalletModal, WalletIcon } from "@solana/wallet-adapter-react-ui";
+import { useWallet } from "@solana/wallet-adapter-react";
+import { Button } from "@components/Button";
+import { Icon } from "@components/Icon";
+
+export const WalletButton = () => {
+	const { wallet, publicKey, disconnect } = useWallet();
+	const { setVisible } = useWalletModal();
+	const [dropdownVisible, setDopdownVisible] = useState(false);
```
Comment (RuvendeGroote) [inline src/components/WalletButton.tsx]: Dopdown

**Comment by RuvendeGroote** [comment]
image
image
there is some misalignment here

**Comment by RuvendeGroote** [comment]
one commit has 'cleanup' as tag. this is not specified in our git guidelines. maybe we can use refactor here?
if we feel like as a team we need additional tags we can always discuss it of course. I do want to prevent a jungle of commit tags as that would defeat part of the purpose

**Review by RuvendeGroote — CHANGES_REQUESTED** [review]

**Comment by RuvendeGroote** [comment]
your latest feat commit is a good example of something that could be included in another commit with rebase interactive

**Comment by BartDeCaluwe** [comment]
> one commit has 'cleanup' as tag. this is not specified in our git guidelines. maybe we can use refactor here? if we feel like as a team we need additional tags we can always discuss it of course. I do want to prevent a jungle of commit tags as that would defeat part of the purpose

Yes, my bad. I don't think `refactor` is a good candidate in that scenario. No functionality was rewritten, only some unused css was removed. 
Unless we agree that we bundle cleanup under the `refactor` tag which is fine by me.

**Review by RuvendeGroote — APPROVED** [review]

**Review by BouweCeunen — APPROVED** [review]

**Pull request merged**